doc/snippets/Microsoft.Data.SqlClient/SqlConnectionIPAddressPreference.xml

```diff
@@ -0,0 +1,28 @@
+<docs>
+    <members name="SqlConnectionIPAddressPreference">
+        <SqlConnectionIPAddressPreference>
+            <summary>
+                Specifies a value for IP address preference during a TCP connection.
```

Comment:
Please add a different section for .NET Framework covering TNIR in remarks.
As TNIR also needs to disabled to bring this property into effect.

You can add 2 XML elements instead of 1 and refer them respectively:
SqlConnectionIPAddressPreferenceNetCore (as-is)
SqlConnectionIPAddressPreferenceNetFx (includes TNIR mention in remarks)